apps/external-devices/device-descriptors/GPS_NMEA.json

```diff
@@ -0,0 +1,13 @@
+{
+    "deviceVendor": "Generic",
+    "deviceName": "GPS (NMEA)",
```

Comment:
specify somewhere here (in the name?) that it's as a tracker, and that it's a generic serial protocol?